src/core/dev/cq_mgr_rx_strq.cpp

```diff
@@ -84,7 +84,10 @@ cq_mgr_rx_strq::~cq_mgr_rx_strq()
         cq_logdbg("Clearing %zu stride objects)", m_rx_queue.size());
 
         while (!m_rx_queue.empty()) {
-            reclaim_recv_buffer_helper(m_rx_queue.get_and_pop_front());
+            auto buff = m_rx_queue.get_and_pop_front();
```

Comment:
Please use auto only if the type can be deducted from the rest of the line.
Otherwise i prefer not to use auto unless its has meaning.

Reply:
Fixed